Sources/Tracing/Span.swift

```diff
@@ -329,6 +329,10 @@ extension SpanAttributes {
     public func forEach(_ callback: (String, SpanAttribute) -> Void) {
         self._attributes.forEach { callback($0.key, $0.1) }
     }
+    
+    public var keys: Dictionary<String, SpanAttribute>.Keys {
+        return self._attributes.keys
```

Comment:
I always get caught by this too 😁

Reply:
Documentation comment please though.

Okey to expose this I think